scene/gui/color_picker.cpp

```diff
@@ -398,25 +429,42 @@ void ColorPicker::_copy_hsv_to_color() {
 	}
 }
 
+void ColorPicker::_select_from_preset_container(const Color &p_color) {
+	if (selected_preset) {
+		selected_preset->set_pressed(false);
+		selected_preset = nullptr;
+	}
+
+	for (int i = 1; i < preset_container->get_child_count(); i++) {
+		ColorPresetButton *current_btn = Object::cast_to<ColorPresetButton>(preset_container->get_child(i));
+		if (current_btn && p_color == current_btn->get_preset_color()) {
+			current_btn->set_pressed(true);
+			selected_preset = current_btn;
+			break;
+		}
+	}
+}
+
 ColorPicker::PickerShapeType ColorPicker::_get_actual_shape() const {
 	return modes[current_mode]->get_shape_override() != SHAPE_MAX ? modes[current_mode]->get_shape_override() : current_shape;
 }
 
 void ColorPicker::_reset_theme() {
-	Ref<StyleBoxEmpty> style_box_empty(memnew(StyleBoxEmpty));
-
+	Ref<StyleBoxFlat> style_box_flat(memnew(StyleBoxFlat));
+	style_box_flat->set_default_margin(SIDE_TOP, 16 * get_theme_default_base_scale());
+	style_box_flat->set_bg_color(Color(0.2, 0.23, 0.31).lerp(Color(0, 0, 0, 1), 0.3).clamp());
```

Comment:
In addition to **RAW** mode, the slider will be overwritten by the drawn texture. It doesn't seem necessary.
```suggestion
```